apps/mobile/src/common/global-sheet-provider.tsx

```diff
@@ -0,0 +1,29 @@
+import { createContext, useContext, useRef } from 'react';
+
+import { HasChildren, SheetRef } from '@leather.io/ui/native';
+
+export interface GlobalSheetContextValue {
```

Comment:
How about not labelling this as `Global` and just calling it `Sheet`?

Reply:
It seems like this is the central point for sheet through which all sheets need to go? 

Maybe I'm mis-understanding though and there will be more localised `Sheet`s also. 

I'm just suggesting to call it `sheet` instead of `global-sheet` and use it's filesystem placement to infer it's global.

Reply:
That's the nuance, not *all* sheets–just global sheets, hence the name.

This isn't a new implementation, but simply renaming existing implementation which used to be called sheet-navigator and co-located with SheetNavigationContainer–misleading because this has no overlap with navigation.
I propose we keep it as is within this PR, and revisit within [LEA-2380: Improve global sheet handling in the mobile app](https://linear.app/leather-io/issue/LEA-2380/improve-global-sheet-handling-in-the-mobile-app)